### PR TITLE
Portfolie Mobile version Last section completed

### PR DESCRIPTION
**Pull request by ademibowale**
Portfolio: mobile version
Learning objectives
Understand how to parse a Figma design to create a UI.
Use Flexbox to place elements in the page.
Use CSS Grid to place elements in the page.
Build a personal portfolio site.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+                    <img src="images/vector.png90.png" class="social-logo logon" alt="my logo">
+                </div>
+            </li>
+            <li><img src="images/vector.png8.png" alt="my logo" class="social-logo"></li>
+            <li><img src="images/vector.png4.png" alt="my logo" class="social-logo"></li>
+            <li><img src="images/vector.png2.png" alt="my logo" class="social-logo"></li>
+        </ul>
+    </section>
+    <section class="work-section">
+        <h2 class="works">My Recent Works</h2>
```
Comment (prolajumokeoni) [inline index.html:43]: Figma Design          |  Your design     
:-------------------------:|:-------------------------:
Screenshot_75  |  Screenshot_74

- [ ] Kindly add the horizontal line after my recent works heading 👍🏿

Comment (prolajumokeoni) [inline index.html:43]: Figma Design          |  Your design     
:-------------------------:|:-------------------------:
Screenshot_75  |  Screenshot_74

- [ ] Kindly add the horizontal line after my recent works heading 👍🏿

File: index.html [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+                    <img src="images/vector.png" class="social-logo logon" alt="my logo">
+                    <img src="images/vector.png90.png" class="social-logo logon" alt="my logo">
+                </div>
+            </li>
+            <li><img src="images/vector.png8.png" alt="my logo" class="social-logo"></li>
+            <li><img src="images/vector.png4.png" alt="my logo" class="social-logo"></li>
+            <li><img src="images/vector.png2.png" alt="my logo" class="social-logo"></li>
+        </ul>
+    </section>
+    <section class="work-section">
```
Comment (prolajumokeoni) [inline index.html:42]: - [ ] Kindly add the add button styling: Active, Hover and Pressed (see screenshot)

Screenshot_76

- Please make this adjustment for all the  buttons in this section👍🏿

File: index.html [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+                <button class="button card-button" type="button">See Project</button>
+            </li>
+        </ul>
+
+    </section>
+    <section class="about-section">
+        <h2 class="about-title">About me</h2>
+        <div class="about">
+            <p class="about-text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <button class="button about-button" type="button">Get My Resume</button>
```
Comment (prolajumokeoni) [inline index.html:142]: - [ ] Kindly add the add button styling: Active, Hover and Pressed (see screenshot)

Screenshot_76

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @ademibowale, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 
-  I advise you use the same repository and different pull request for each milestone, this will help the reviewer understand the pr better, please use the same pull request for re review, when it is approved the you can merge and move to the next milestone  on a new pr👍🏿  
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codepantha — APPROVED** [review]
Hi @ademibowale 👋,

## STATUS: APPROVED ✔️  

Great work on making the changes requested by the previous reviewer 👏 
Your project is now complete.



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**